min-image/runtimes/python/server.py

```diff
@@ -208,6 +208,7 @@ def main():
             file.write(pid)
             print(f'server.py: joined cgroup, close FD {fd_id}')
 
+    # drop privileges here
```

Comment:
Can we make capability dropping optional, via config, like it is for seccomp?  https://github.com/open-lambda/open-lambda/blob/main/min-image/runtimes/python/server.py#L189

Reply:
@tylerharter I added the command line argument for enabling capabilities as well. However, we'll need to pass the flag on to everywhere we drop capabilities - to check it and drop only if it's true. Wondering what would be the best way to do that.